placesautocomplete/src/main/java/com/seatgeek/placesautocomplete/PlacesAutocompleteTextView.java

```diff
@@ -171,18 +170,20 @@ public void onItemClick(final AdapterView<?> parent, final View view, final int
                 }
             }
         });
-        if(clearEnabled)
-            enableClearButton(true);
+        if(clearEnabled) {
+            enableClearButton(context, true);
+        }
         super.setDropDownBackgroundResource(R.drawable.pacv_popup_background_white);
     }
 
-    private void enableClearButton(boolean value){
+    private void enableClearButton(Context context, boolean value){
```

Comment:
You should be able to avoid passing the context in by using the `getContext()` method on View (which this class extends)

Reply:
yea your right